app/decorators/event_decorator.rb

```diff
@@ -0,0 +1,9 @@
+# :reek:NilCheck
+
+class EventDecorator < Draper::Decorator
+  delegate_all
+
+  def attendance?(user_id)
```

Comment:
this is confusing naming. when i'm reading `event.decorate.attendance?` I'm asking myself "Is event an attendance?" but this is wrong, event and attendance are different entities. I believe you understand this, so please choose better name